plugins/guests/linux/cap/public_key.rb

```diff
@@ -53,6 +53,7 @@ def self.remove_public_key(machine, contents)
               result=$?
             fi
             rm -f '#{remote_path}'
+            chmod 0600 ~/.ssh/authorized_keys
```

Comment:
It is not reflected even in the latest version(v1.9.7) of the feeling I saw.Is this okay with this?
[here](https://github.com/mitchellh/vagrant/blob/v1.9.7/plugins/guests/linux/cap/public_key.rb#L54)
Sorry for my misunderstanding.